pkg/otlp/metrics/metrics_translator.go

```diff
@@ -40,7 +40,21 @@ const (
 	errNoBucketsNoSumCount string = "no buckets mode and no send count sum are incompatible"
 )
 
-var _ source.Provider = (*noSourceProvider)(nil)
+var (
+	_ source.Provider = (*noSourceProvider)(nil)
+
+	rateAsGaugeMetrics = map[string]string{
```

Comment:
Typically if what you want to represent is a set, you can use a `map[string]struct{}` and check 

```go
if _, ok := rateAsGaugeMetrics[dims.name]; ok {
   // do something
}

```